plugins/modules/web_infrastructure/apache2_module.py

```diff
@@ -49,6 +49,10 @@
      type: bool
      default: False
 requirements: ["a2enmod","a2dismod"]
+notes:
+  - This does not work on RedHat-based distributions.
+  - It does work on Debian- and SuSE-based distributions.
+  - Whether it works on others depend on whether the C(a2enmod) and C(a2dismod) tools are available or not.
```

Comment:
```suggestion
  - This does not work on RedHat-based distributions. It does work on Debian- and SuSE-based distributions.
    Whether it works on others depend on whether the C(a2enmod) and C(a2dismod) tools are available or not.
```
I would avoid putting things that belong together into different notes. First this makes it harder to understand, since there's some larger amount of space between every sentence now, and also if you (or someone else) ever add a docs fragment which contains notes, ansible will sort the notes alphabetically after merging the lists. Then these sentences will appear in another order, perhaps with other sentences inbetween, and make less sense.